src/search/pdbs/pattern_database.cc

```diff
@@ -239,6 +248,21 @@ void PatternDatabase::create_pdb(
         }
     }
 
+    if (compute_plan) {
+        /*
+          If computing a plan during Dijkstra, we store, for each state,
+          an operator leading from that state to another state on a
+          strongly optimal plan of the PDB. We store the first operator
+          encountered during Dijstra and only update it if the goal distance
+          of the state was updated or, in the presence of zero-cost operators,
+          the state could be reached with the same cost but with fewer
+          zero-cost operators. For the latter, we need to count the number of
+          zero-cost operators leading to each state.
+         */
+        generating_op_ids.resize(num_states);
+        number_of_zero_cost_ops.resize(num_states);
```

Comment:
I'd store the info in a single vector of two-int structs: better cache-locality, less memory, maybe simpler code.